README.md

```diff
@@ -75,14 +75,16 @@ Overall it tries to give you hints regarding why it isn't doing what you expecte
 - "kaka" is swedish for "cookie"
 
 # TL;DR
-
 Open a terminal or SSH on the server running Plex Media Center
+## First install
+```
+git clone https://github.com/mrworf/plexupdate.git
+chmod +x plexupdate/plexupdate.sh
```

Comment:
You actually get the chmod +x for free when you use git, so this line can be skipped.